src/openforms/products/static_variables/static_variables.py

```diff
@@ -0,0 +1,29 @@
+from typing import TYPE_CHECKING, Optional
+
+from django.utils.translation import gettext_lazy as _
+
+from openforms.variables.base import BaseStaticVariable
+from openforms.variables.constants import FormVariableDataTypes
+from openforms.variables.registry import register_static_variable
+
+if TYPE_CHECKING:  # pragma: nocover
+    from openforms.submissions.models import Submission
+
+
+@register_static_variable("product")
+class Product(BaseStaticVariable):
+    name = _("Product")
+    data_type = FormVariableDataTypes.object
+
+    def get_initial_value(self, submission: Optional["Submission"]) -> Optional[dict]:
+        if not submission or not submission.form.product:
+            return None
+
+        product = submission.form.product
+        data = {
+            "uuid": product.uuid,
+            "name": product.name,
+            "price": product.price,
+            "information": product.information,
+        }
```

Comment:
Maybe it would be nice to use a serializer to serialise the data to a dict

Reply:
Isn't serialization the process of converting data types like `dict` to a string (e.g. JSON)? If `dict` is the correct type of return value for `get_initial_value`, what else is there to do?

Reply:
It works, but all attributes of `product` are transformed into strings. In particular, `product.price` will be (for example) `'9.99'` instead of `9.99`. Is this intended?

Reply:
I see that we don't override `COERCE_DECIMAL_TO_STRING`, and looking at the documentation of the decimal field (https://www.django-rest-framework.org/api-guide/fields/#decimalfield), it seems that it is true by default.

Reply:
doing math on floats, especially when money is involved, is a bad idea due to floating point rounding errors.

It'd make more sense to expose the price as an integer, in cents and then divide that back by 100 for the final result.